server/routes/tiering-additional-information.ts

```diff
@@ -0,0 +1,7 @@
+import { Router } from 'express'
+
+export default function tierAddInfoRoutes(router: Router) {
+  router.get('/tiering-additional-information', (req, res) => {
+    res.render('/tiering-additional-information')
```

Comment:
You'll still need this one to be `/pages/info` - the first string is the URL path, the second is the path to your .njk file.

Reply:
Took that out by accident 😬